justdail_scraper.py

```diff
@@ -0,0 +1,42 @@
+from bs4 import BeautifulSoup
+import urllib
+import csv
+import constant
+
+
+def getDetails(body):
+	name= body.find('span', {'class':constant.Name_Class}).a.string
+	phone=body.find('p', {'class':constant.Contact_Class}).span.a.string
+	address=body.find('span', {'class':constant.Address_Class}).text.strip()
+	dict_service = {}
+	if name:
```

Comment:
Why have an if condition here? Your dict will have an inconsistent structure this way, sometimes the dict will have some keys and sometimes it won't. 

Will cause a problem in the other functions that are using the output of this function. 

Have a dict with the keys already set to empty/None values